### DISCUSSION TITLE
[Projects Beta] Automatically add 'date added' field

### DISCUSSION THREAD

**Discussion by sylviepaquette** [community/community#7280] [Projects and Issues]
Currently I had to create a field, and then every time an issue is added to the project, I have to manually add the date added. I do this in order to create a view where I can see which team member has been working on which issues recently.

**Comment by dillingham** [comment]
I would also love the `created_at` field, also `updated_at` and `activity_at`

Being able to sort by recently replied would be very useful

**Comment by ianriosbaf** [comment]
If an addition to this github action https://github.com/actions/add-to-project would help, I imagine it wouldn't be too hard to update the action to also fill out a field in the project for `created_at` - I just don't have time to do it myself :(
